testify.go

```diff
@@ -77,7 +99,50 @@ func (st sinkTestingT) Errorf(format string, args ...interface{}) {
 //////////////////////////////////////////////////////////////////////////////
 // Assertions
 
-// Assertions provides the same interface as the assert.Assertions struct.
+// Assertions provides helpers to assert conditions in crossdock behaviors.
+//
+// All assertions can include informative error messages formatted using
+// fmt.Sprintf style,
+//
+// 	assert := Assert(t)
+// 	assert.Contains(foo, "bar", "expected to find 'bar' in %q", foo)
+//
+// All assert operations return true if the condition was met and false
+// otherwise. This allows gating operations that would otherwise panic behind
+// preconditions.
+//
+// 	if assert.Error(t, err, "expected failure") {
+// 		assert.Contains(t, err.Error(), "something went wrong", "error message mismatch")
+// 	}
+//
+// Additionally, in case of failure, all Assertions make an attempt to
+// provide a stack trace in the error message.
+//
+// Four kinds of Assertions objects are offered via the corresponding
+// functions:
+//
+// Assert(T): All asserts will result in a success or failure being logged to the
+// crossdock.T. Execution will continue on failure.
+//
+// Checks(T): Only failures will be logged to crossdock.T. Execution will
+// continue on failure.
+//
+// Require(T): All asserts will result in a success or failure being logged to
+// the crossdock.T. Execution of the behavior will be terminated immediately
+// on failure.
+//
+// Fatals(t): Only failures will be logged to crossdock.T. Execution of the
+// behavior will be temrinated immediately on failure.
+//
+//	                     +--------+--------+---------+--------+
```

Comment:
this table doesn't look very aligned:
![image](https://cloud.githubusercontent.com/assets/140159/17304618/6031605c-57db-11e6-8325-fa743bd052cb.png)